File: primerTrimestre/java basico/1secuenciales/Ej10sec.java
/** Un alumno desea saber cual será su calificación final en la materia de Algoritmos. Dicha calificación
 * se compone de los siguientes porcentajes:
 * 55% del promedio de sus tres calificaciones parciales.
 * 30% de la calificación del examen final.
 * 15% de la calificación de un trabajo final.
 * 
 * @author Antonio Quesada
 */


import java.util.Scanner;

public class Ej10sec {
  public static void main(String[] args) {
    Scanner s = new Scanner(System.in);
    
    //Definimos las variables.
    double parcial1;
    double parcial2;
    double parcial3;
    double examenFinal;
    double trabajo;
    double notaFinal;
    
    //Petición de datos.
    System.out.println("Inserte la calificación del primer parcial:");
    parcial1 = s.nextDouble();
    System.out.println("Inserte la calificación del segundo parcial:");
    parcial2 = s.nextDouble();
    System.out.println("Inserte la calificación del tercer parcial:");
    parcial3 = s.nextDouble();
    System.out.println("Inserte la calificación del examen final:");
    examenFinal = s.nextDouble();
    System.out.println("Inserte la calificación del trabajo:");
    trabajo = s.nextDouble();
    
    //Cálculo de la nota final.
    notaFinal = ((parcial1+parcial2+parcial3)/3)*0.55+0.3*examenFinal+0.15*trabajo;
    
    //Mostramos la nota final.
    System.out.println("La calificación final será "+notaFinal);
 

  }

}

File: primerTrimestre/introduccion_programacion_python/1secuenciales_py/ej10sec.py
# Programa: ej10sec.py
# Propósito: Un alumno desea saber cual será su calificación final en la materia de Algoritmos. 
#Dicha calificación se compone de los siguientes porcentajes:
#55% del promedio de sus tres calificaciones parciales.
#30% de la calificación del examen final.
#15% de la calificación de un trabajo final.
#
# Autor: Antonio Quesada
# Fecha: 13/10/2019.
#
# Variables a usar:
#   * parcial1
#   * parcial2
#   * parcial3
#   * examen_final
#   * trabajo
#
# Algoritmo:
#   LEER parcial1, parcial2, parcial3, examen_final, trabajo
#   nota_final <-- ((parcial1+parcial2+parcial3)/3)*0.55+0.3*examen_final+0.15*trabajo)
#   ESCRIBIR nota_final


# Petición de datos
parcial1 = float(input("Inserte la calificación del primer parcial: "))
parcial2 = float(input("Inserte la calificación del segundo parcial: "))
parcial3 = float(input("Inserte la calificación del tercer parcial: "))
examen_final = float(input("Inserte la calificación del examen final: "))
trabajo = float(input("Inserte la calificación del trabajo: "))
print("---------------------------------------------------------------")

#Cálculos
nota_final = ((parcial1+parcial2+parcial3)/3)*0.55+0.3*examen_final+0.15*trabajo

#Cálculos y Salida
print ("Su calificación final será de", nota_final)





	
	

	






	
	

	
